packages/ember-simple-auth-cookie-store/lib/simple-auth-cookie-store/stores/cookie.js

```diff
@@ -181,5 +208,58 @@ export default Base.extend({
       Ember.run.cancel(this._syncDataTimeout);
       this._syncDataTimeout = Ember.run.later(this, this.syncData, 500);
     }
+  },
+
+  /**
+    @method pageVisibilityInit
+    @private
+  */
+  pageVisibilityInit: function(){
+    var stateKey;
+    var eventKey;
+    var keys = {
+      hidden: "visibilitychange",
+      webkitHidden: "webkitvisibilitychange",
+      mozHidden: "mozvisibilitychange",
+      msHidden: "msvisibilitychange"
+    };
+    for (stateKey in keys) {
+      if (stateKey in document) {
+        eventKey = keys[stateKey];
+        break;
+      }
+    }
+    return function(c) {
+      if (c) {document.addEventListener(eventKey, c);}
+      return !document[stateKey];
+    };
+  },
```

Comment:
@marcoow : I'm following this [post](http://stackoverflow.com/questions/19519535/detect-if-browser-tab-is-active-or-user-has-switched-away/19519701#19519701) you can find all details you need with a demo for it.
@drewcovi : No, It will not break in IE9  and it's supporting IE10+